target/snitch_cluster/sw/apps/snax-data-reshuffler/data/datagen.py

```diff
@@ -208,8 +208,8 @@ def emit_gemm_data(**kwargs):
             padded_output_tensor_w * padded_output_tensor_h * kwargs["Cin"]
         )
 
-        assert padded_output_tensor_w == kwargs["W"]
-        assert padded_output_tensor_h == kwargs["H"]
+        assert padded_output_tensor_w % 8 == 0
+        assert kwargs["Cin"] % 8 == 0
```

Comment:
This is because we are limited to the max bandwidth of 512 bits right?

Reply:
yes